app/templates/frameworks/start_declaration.html

```diff
@@ -0,0 +1,51 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}
+  {% block page_title_inner %}
+    Make your supplier declaration
+  {% endblock %} – Digital Marketplace
+{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace",
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": "Your account",
+      },
+      {
+        "link": url_for(".framework_dashboard", framework_slug=framework.slug),
+        "label": "Apply to " + framework.name,
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+  <div class="grid-row framework-dashboard">
+    <div class="column-two-thirds">
+      {% with
+         heading = self.page_title_inner(),
+         smaller = True
+      %}
+        {% include "toolkit/page-heading.html" %}
+      {% endwith %}
+
+      <p>You must answer all questions in the declaration to confirm you meet the minimum standards for providing digital services to government.</p>
+      <br>
+      <p>The Crown Commercial Service will review your answers. If you don’t meet the minimum standards, your application won’t be successful.</p>
+      <br>
+      <p>You can come back and change your answers at any time before the framework application deadline, 5pm BST, 23 June 2016.</p>
```

Comment:
The date here should come from the `framework_close_date` in the framework content, which you should be able to get at in the view the same time you get the "next section" name (have a look at existing code, you might need to `load_messages('dates')` or something like that).